man/logind.conf.xml

```diff
@@ -200,6 +200,7 @@
         <term><varname>HandleLidSwitchExternalPower=</varname></term>
         <term><varname>HandleLidSwitchDocked=</varname></term>
         <term><varname>HandleRebootKey=</varname></term>
+        <term><varname>HandleRebootKeyLongPress=</varname></term>
```

Comment:
hmm what about the same for the three other keys? my guess is that people whose device has only one suspend key might want to bind a quick press to suspend and a long press to hibernate or so...

Reply:
Agree. I will update PR.

Reply:
But let's straighten the implementation details for this particular key first.

Reply:
Or may be push the reboot key change first and then I will make separate PR for supporting other keys? Anyway that code for other keys will be mostly copy-paste of this one(if you have no better ideas).

Reply:
@poettering 
Hi! I added the long press implementation for the other 3 keys, can you review the changes please?